pkg/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+// Config is the global configuration for the training operator.
+var Config struct {
+	PyTorchInitContainerTemplateFile string
+	PyTorchInitContainerImage        string
+}
+
+const (
+	// PyTorchInitContainerImageDefault is the default image for the pytorch
+	// init container.
+	PyTorchInitContainerImageDefault = "alpine:3.10"
```

Comment:
I see some other similar cases use `busybox` but I didn't get a chance to check difference? Any clues?

Reply:
is image size the only dimension?

Reply:
I think so. Image size is the only thing I can think of.